.circleci/config.yml

```diff
@@ -114,11 +114,18 @@ jobs:
       - checkout
       - attach_workspace:
           at: /rest
+      - run:
+          command: |
+            suffix=$(if [ <<parameters.basedir>> = "/rest" ]; then echo "REST"; else echo "AIO"; fi)
+            lower=PYPI_TOKEN_GCLOUD_$(echo $suffix)_<<parameters.cwd>>
+            upper=$(echo $lower | awk '{ print toupper }')
+            export POETRY_PYPI_TOKEN_PYPI=$(printenv | grep $upper | awk -F= '{ print $2 }')
```

Comment:
Exporting this variable to the shell env will not persist beyond this `run` block. If it needs to persist into the next `run` blocks, as it should for the `poetry publish` step, then it needs to be pushed into the shell dotfile via `BASH_ENV`:

https://circleci.com/docs/set-environment-variable/#set-an-environment-variable-in-a-shell-command